frontend/src/pages/TermPlanner/ImportPlannerMenu/ImportPlannerMenu.tsx

```diff
@@ -79,7 +79,16 @@ const ImportPlannerMenu = () => {
           dispatch(updateDegreeLength(fileInJson.numYears));
           dispatch(updateStartYear(fileInJson.startYear));
           if (planner.isSummerEnabled !== fileInJson.isSummerEnabled) {
-            dispatch(toggleSummer());
+            try {
+              axios.put('/user/toggleSummerTerm', {}, { params: { token } });
+            } catch {
+              openNotification({
+                type: 'error',
+                message: 'Error setting summer term',
+                description: 'An error occurred when toggling the summer term.'
+              });
+              return;
+            }
```

Comment:
Make function async and await the request. 

Also since this notification is for importing a file to change settings I recommend something more specific to this use case for the description e.g. `An error occurred when trying to import summer term visibility` or something along those lines.

Reply:
Fixed